cmd/build/helmify/delete-merge-values.yaml

```diff
@@ -15,6 +15,11 @@ spec:
               $patch: delete
             - containerPort: 9090
               $patch: delete
+          volumeMounts:
+            - mountPath: /certs
+              name: cert
+              readOnly: true
+              $patch: delete
```

Comment:
I added these here to avoid duplicate entries when generating manifests. I renamed the file as it is no longer only deleting ports.